release notes/v0.36.0.md

```diff
@@ -0,0 +1,76 @@
+k6 v0.36.0 is here! 🎉 It introduces a couple new features which enhance its usability, and includes a bunch of fixes and the result of refactoring ongoing efforts.
+
+In total, we have closed 27 issues.
+
+## New Features!
+
+### Sourcemaps support ([#2082](https://github.com/grafana/k6/pull/2082))
+
+// TODO: Ask @MStoykov for support on this
+
+### Ability to abort tests ([#2093](https://github.com/grafana/k6/pull/2093))
+
+Thanks to the contribution of [@gernest](https://github.com/gernest), k6 now has the ability to abort a test. [#2093](https://github.com/grafana/k6/pull/2093) made the `k6/execution` exposes the `test.abort()` function, which allows to immediately abort a k6 load test run with the exit code `108`.
+
+Aborting is possible during initialization:
+```javascript
+import exec from "k6/execution";
+exec.test.abort();
+```
+
+As well as inside the default function:
+```javascript
+import exec from "k6/execution";
+
+export default function() {
+  // Note that you can abort with a specific message too
+  exec.test.abort("this is the reason");
+}
+
+export function teardown() {
+  console.log("teardown will still be called after test.abort()");
+}
+```
+
+Or during the VU initialization:
+```javascript
+import exec from "k6/execution";
+
+// This won't fail on initial parsing of the script, but on VU initialization.
+if (__VU == 1) {
+  exec.test.abort();
+}
+
+export default function() {}
+```
+
```

Comment:
It might make sense to add a note about cloud execution too: ~ `Aborting the test is also fully supported in k6 Cloud, keeping the same UX for both local and cloud users`. AFAIK, it _was_ a major part of the work there :smile: